benchmarks/device_uvector/device_uvector_bench.cu

```diff
@@ -38,7 +39,8 @@
 void BM_UvectorSizeConstruction(benchmark::State& state)
 {
   rmm::mr::cuda_memory_resource cuda_mr{};
-  rmm::mr::pool_memory_resource<rmm::mr::cuda_memory_resource> mr{&cuda_mr};
+  rmm::mr::pool_memory_resource<rmm::mr::cuda_memory_resource> mr{
+    &cuda_mr, rmm::percent_of_free_device_memory(1. / 2)};
```

Comment:
**question** This is asking for an initial pool size of `0%` since there is no `rmm::percent_of_free_device_memory(double fraction)` and the numeric promotion turns 0.5 into 0 when resolving to `rmm::percent_of_free_device_memory(int percent)`.

Is that what you want, or did you mean:

```suggestion
    &cuda_mr, rmm::percent_of_free_device_memory(50)};
```

Reply:
Yes, this is a vestige of a past version. Thanks for catching.